src/main/resources/application.yml

```diff
@@ -30,16 +30,26 @@ spring:
           starttls:
             enable: true
     confirmLinkIP: http://localhost:4200/
+  servlet:
+    multipart:
+      max-file-size: 50000KB
+      max-request-size: 50000KB
 jwt:
   bearer: Bearer
   # those are the expiration durations for jwt access and jwt refresh
   # introduced in milliseconds.
-  accessExpirationMs: 18000000
-  refreshExpirationMs: 30000000
+  accessExpirationMs: 800000000
+  refreshExpirationMs: 900000000
 
 springdoc:
   swagger-ui:
     path: /swagger-ui.html
 
+path:
+  basePackagePath: "C:/Users/Arthur_Khachatryan/avatars"
```

Comment:
Not sure that all users have the same path. Use relative path.

Reply:
In my case when I am running the app on local I have access only to this filder,  but when I am creating the docher inage I change this .